solution-base/mongodb/charts/mongodb/templates/poddisruptionbudget-arbiter-rs.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if and (and .Values.replicaSet.enabled .Values.replicaSet.pdb.enabled) (gt .Values.replicaSet.replicas.arbiter 0) }}
-apiVersion: policy/v1beta1
+apiVersion: policy/v1
```

Comment:
Why not using an include here as well ?

Reply:
the included-file is in the `mongodb-sharded` chart, but not in the `mongodb` chart